modules/persistenceService/src/main/resources/db/migration/V2__ALTER_USER_TABLE_DROP_ADD_NEW_COLUMN.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE userdb
+DROP COLUMN username,
+DROP COLUMN name,
+ADD COLUMN slack_user_id uuid UNIQUE NOT NULL,
+ADD COLUMN slack_channel_id uuid UNIQUE NOT NULL;
+
+CREATE TABLE repositories(
+    repository_id uuid PRIMARY KEY NOT NULL,
+    owner VARCHAR NOT NULL,
+    repository VARCHAR NOT NULL
```

Comment:
I  would suggest here:
```sql
ADD CONSTRAINT unique_repos UNIQUE (owner, repository);
```
This way you save yourself from having duplicate repos

Reply:
Thank you 😊